plasma_framework/contracts/mocks/exits/payment/routers/PaymentInFlightExitRouterMock.sol

```diff
@@ -19,11 +20,11 @@ contract PaymentInFlightExitRouterMock is PaymentInFlightExitRouter {
         inFlightExitMap.exits[exitId].exitMap = Bits.setBit(inFlightExitMap.exits[exitId].exitMap, 255);
     }
 
-    function getInFlightExitInput(uint192 exitId, uint8 inputIndex) public view returns (PaymentOutputModel.Output memory) {
+    function getInFlightExitInput(uint192 exitId, uint8 inputIndex) public view returns (PaymentExitDataModel.WithdrawData memory) {
```

Comment:
minor: for `inputIndex ` and `outputIndex` below, probably bump it to uint16 to be consistent with our current index size.

Reply:
done